### ISSUE TITLE
Widescreen?

### ISSUE THREAD

**Issue by Anonymous941** [Alexyo21/twl_firm_patcher#3]
Hello,

First of all, thank you SO much for making this, I've been waiting for years for someone to finally make EmuNAND usable!  Is it possible to also have widescreen enabled?  I think that the CustomLuma3DS' Quick Switchers have something to do with this, but they all show up as "unknown", despite me having put `TwlBg.cxi` in the `/luma/sysmodules` folder

**Comment by Alexyo21** [comment]
Hi, thanks and I really appreciate it but I'm just one who took this project out to the light again, so all the credits don't go to me.
Talking about the widescreen yeah it's not supported yet with this, but  technically if you use twilightmenu you can place one twlbg with only that widescreen feature and it will combine the patches( you can make one with twpatcher by sono, and you find it on gbatemp), though I cannot tell if it will work cause my emu broke and I have one to test so you will have to wait a while til implement this here,  sorry.
For use obviously you have to have a full emu on sd not rednand.
If you have more doubt I'm here

**Comment by Anonymous941** [comment]
> Hi, thanks and I really appreciate it but I'm just one who took this project out to the light again, so all the credits don't go to me.

Yeah, I've seen https://github.com/RocketRobz/twl_firm_patcher_sdread, but I couldn't figure out how to get it to work :)

> Talking about the widescreen yeah it's not supported yet with this, but I technically if you use twilightmenu you can place one twlbg with only that widescreen feature and it will combine the patches( you can make one with twpatcher by sono, and you find it on gbatemp)

Could I combine them manually?  My idea is to make a .ips file from the .cxi file you compiled, then apply that to widescreen.cxi

**Comment by Alexyo21** [comment]
https://wiki.ds-homebrew.com/twilightmenu/playing-in-widescreen
Here you the guide try this

**Comment by Alexyo21** [comment]
Also you can try combine manually but I do not guarantee results cause the patches can be overwritten
